Framework/Algorithms/src/Minus.cpp

```diff
@@ -33,12 +33,14 @@ void Minus::performBinaryOperation(const HistogramData::Histogram &lhs,
                                    HistogramData::HistogramE &EOut) {
   using std::placeholders::_1;
   std::transform(lhs.y().begin(), lhs.y().end(), YOut.begin(),
-                 std::bind(std::minus<double>(), _1, rhsY));
+                 [rhsY](const double &l) { return l - rhsY; });
   // Only do E if non-zero, otherwise just copy
-  if (rhsE != 0)
-    std::transform(lhs.e().begin(), lhs.e().end(), EOut.begin(),
-                   std::bind(VectorHelper::SumGaussError<double>(), _1, rhsE));
-  else
+  if (rhsE != 0) {
+    double rhsE2 = rhsE * rhsE;
+    std::transform(
+        lhs.e().begin(), lhs.e().end(), EOut.begin(),
+        [rhsE2](const double &l) { return std::sqrt(l * l + rhsE2); });
```

Comment:
Nit pick: we should do `double l` rather than `const double &l`, since the former can use a register and the latter has a potential memory alias (the compiler should optimise away, but this makes it safe)